stellar-multitenant/internal/httphandler/tenants_handler_test.go

```diff
@@ -577,20 +649,62 @@ func Test_TenantHandler_Patch(t *testing.T) {
 			"is_default": false,
 		`
 
-		runSuccessfulRequestPatchTest(t, r, ctx, dbConnectionPool, handler, reqBody, expectedRespBody)
+		runSuccessfulRequestPatchTest(t, r, ctx, dbConnectionPool, handler, reqBody, expectedRespBody, nil)
 	})
 
-	t.Run("successfully updates Status of a tenant", func(t *testing.T) {
-		reqBody := `{"status": "TENANT_ACTIVATED"}`
+	t.Run("successfully updates status of a tenant", func(t *testing.T) {
+		reqBody := `{"status": "TENANT_DEACTIVATED"}`
 		expectedRespBody := `
 			"base_url": null,
 			"sdp_ui_base_url": null,
-			"status": "TENANT_ACTIVATED",
+			"status": "TENANT_DEACTIVATED",
 			"distribution_account": "GCTNUNQVX7BNIP5AUWW2R4YC7G6R3JGUDNMGT7H62BGBUY4A4V6ROAAH",
 			"is_default": false,
 		`
 
-		runSuccessfulRequestPatchTest(t, r, ctx, dbConnectionPool, handler, reqBody, expectedRespBody)
+		tntStatus := tenant.ActivatedTenantStatus
+		runSuccessfulRequestPatchTest(t, r, ctx, dbConnectionPool, handler, reqBody, expectedRespBody, &tntStatus)
+	})
+
+	t.Run("unsuccessfully updates status of a tenant - invalid status", func(t *testing.T) {
```

Comment:
💡  nit: how about rename `"unsuccessfully updates..."` to `"cannot update..."`, like you did in line 687?